packages/nodes-base/nodes/Slack/V2/GenericFunctions.ts

```diff
@@ -162,7 +162,7 @@ export function getMessageContent(
 			};
 			break;
 		case 'block':
-			content = jsonParse(this.getNodeParameter('blocksUi', i) as string);
+			content = this.getNodeParameter('blocksUi', i, {}, { ensureType: 'object' }) as IDataObject;
```

Comment:
This isn't ideal, but considering the state of type-safety in this package, I think this might be the most realistic option right now .